ДЗ.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Седона</title>
+</head>
+
+<body>
+
+<hedar>
```

Comment:
Исправь название тега
```suggestion
<header>
```